pom.xml

```diff
@@ -82,62 +82,62 @@
-        <maven-site-plugin.version>3.9.1</maven-site-plugin.version>
+        <maven-site-plugin.version>3.10.0</maven-site-plugin.version>
         <maven-source-plugin.version>3.2.1</maven-source-plugin.version>
         <maven-surefire-plugin.version>3.0.0-M5</maven-surefire-plugin.version>
-        <maven-war-plugin.version>3.3.1</maven-war-plugin.version>
+        <maven-war-plugin.version>3.3.2</maven-war-plugin.version>
         <maven.scm.version>1.11.2</maven.scm.version>
         <nexus-staging-plugin-version>1.6.8</nexus-staging-plugin-version>
-        <wagon-ssh.version>3.3.2</wagon-ssh.version>
+        <wagon-ssh.version>3.5.1</wagon-ssh.version>
         <!-- versions for dependencies -->
         <!-- things we build -->
-        <bt-adm-version>2.7.0</bt-adm-version>
+        <bt-adm-version>2.7.3</bt-adm-version>
         <bt-common-api-version>37.3.0</bt-common-api-version>
         <!-- things we consume -->
         <bt-args4j-version>2.33</bt-args4j-version>
-        <bt-commons-io-version>2.7</bt-commons-io-version>
-        <bt-commons-lang-version>2.6</bt-commons-lang-version>
-        <bt-fastutil-version>8.4.0</bt-fastutil-version>
+        <bt-commons-io-version>2.11.0</bt-commons-io-version>
+        <bt-commons-lang3-version>3.12.0</bt-commons-lang3-version>
+        <bt-fastutil-version>8.5.6</bt-fastutil-version>
         <!-- Remaining at Guava 26 until we decide what to do about
              https://groups.google.com/g/guava-announce/c/Km82fZG68Sw
              https://github.com/google/guava/issues/3320 -->
         <bt-guava-version>26.0-jre</bt-guava-version>
         <bt-hamcrest-version>2.2</bt-hamcrest-version>
-        <bt-httpasyncclient-version>4.1.4</bt-httpasyncclient-version>
-        <bt-httpclient-version>4.5.12</bt-httpclient-version>
-        <bt-httpcore-version>4.4.13</bt-httpcore-version>
-        <bt-httpmime-version>4.5.12</bt-httpmime-version>
-        <bt-icu4j-version>59.1</bt-icu4j-version>
+        <bt-httpasyncclient-version>4.1.5</bt-httpasyncclient-version>
+        <bt-httpclient-version>4.5.13</bt-httpclient-version>
+        <bt-httpcore-version>4.4.15</bt-httpcore-version>
+        <bt-httpmime-version>4.5.13</bt-httpmime-version>
+        <bt-icu4j-version>70.1</bt-icu4j-version>
         <bt-jackson-version>2.11.1</bt-jackson-version>
         <bt-jarchivelib-version>0.9.100-basis</bt-jarchivelib-version>
-        <bt-junit-version>4.13.1</bt-junit-version>
+        <bt-junit-version>4.13.2</bt-junit-version>
         <bt-liblinear-version>2.30</bt-liblinear-version>
         <bt-log4j-version>2.17.1</bt-log4j-version>
-        <bt-metrics-version>3.2.3</bt-metrics-version>
-        <bt-opencsv-version>2.4</bt-opencsv-version>
+        <bt-metrics-version>4.2.7</bt-metrics-version>
+        <bt-opencsv-version>5.5.2</bt-opencsv-version>
         <bt-protobuf-version>3.12.2</bt-protobuf-version>
-        <bt-slf4j-version>1.7.28</bt-slf4j-version>
```

Comment:
COMN-280 indicates that this should be updated to 1.7.32, but I assume 1.7.33 is intentional

Reply:
It is intentional. Version 1.7.33 was released after I’d written the ticket.